### ISSUE TITLE
Running the pipeline without sample list.

### ISSUE THREAD

**Issue by gaworj** [mjmlab/pyinseq#88]
Hi,

Is it possible to run the pyinseq pipeline without sample list? My TnSeq data is already demultiplexed after the sequencing run so I cannot provide additional barcode sequences. 

Bests,
Jan

**Comment by mandel01** [comment]
Can you provide more detail about the format the data are in? In particular,
- What are the file names of the fastq files (are these sample names)?
- What do the sequence data look like? The pyinseq pipeline expects [barcode][16-17 nt chromosome]TA[remaining transposon sequence]. Is that similar but without the barcode?
- Did you use a mariner transposon?

**Comment by gaworj** [comment]
My fastqs have simple names such WT_R1.fastq.gz or BHI_R1.fastq.gz. My samples have been already demultiplexed just after Illumina run using bcl2fastq tool.
I have recently read some papers describing barcoding strategy for TnSeq libraries and our protocol is different.
We have used Nextera XT-like barcoding strategy similar to described herein: https://www.protocols.io/view/transposon-insertion-sequencing-tn-seq-library-pre-rm7vzn6d5vx1/v1
ISS1 transposon was used so it was fully random mutagenesis like Tn5 strategy. 
In summary I suspect that my seq data is not compatible with pyinseq workflow.

**Comment by mandel01** [comment]
You might consider contacting the author of that protocol for additional guidance on data processing.

Additionally, I don't substantial experience with these tools, but you also might look at the Java application [Tn-Seq Explorer](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0126070) or the Python software [TRANSIT](https://transit.readthedocs.io/en/latest/).
